dockstore-integration-testing/src/test/java/io/dockstore/webservice/UserResourceIT.java

```diff
@@ -85,6 +85,23 @@ public void resetDBBetweenTests() throws Exception {
         CommonTestUtilities.cleanStatePrivate2(SUPPORT, false);
     }
 
+    @Test
+    public void testAddUserToOrgs() {
+        ApiClient client = getWebClient(USER_2_USERNAME, testingPostgres);
+        UsersApi userApi = new UsersApi(client);
+        List<Workflow> workflows = userApi.refreshWorkflowsByOrganization((long)1, "DockstoreTestUser");
+
+        // Remove an association with an entry
+        long numberOfWorkflows = workflows.size();
+        testingPostgres.runUpdateStatement("delete from user_entry where entryid = 951");
+        long newNumberOfWorkflows = userApi.userWorkflows((long)1).size();
+        assertEquals("Should have one less workflow", numberOfWorkflows - 1, newNumberOfWorkflows);
+
+        // Add user again
+        newNumberOfWorkflows= userApi.addUserToDockstoreWorkflows().size();
```

Comment:
Should test something about the workflows (follow up on serviceDAO discussion)
e.g. if by some bizarro coincidence this returned an equal number of services vs workflows